boards/sim/sim/sim/README.txt

```diff
@@ -199,6 +199,16 @@ REVISIT: This may not long be an issue even with NSH because of the recent
 redesign of how the stdio devices are handled in the simulation (they should
 no longer freeze the simulation).
 
+Update: Please issue these commands to setup the reliable network on Ubuntu:
+
+  sudo apt-get -y install net-tools
+  sudo nuttx/tools/simbridge.sh eth0 on
+
+Here is some tips you may need:
+  1.Must launch the exectuable with the root permission
+  2.Have to use virtual machine if host is in corporation network
```

Comment:
why? just because it's better to isolate dev network?

Reply:
In my case, the coperation network enable some adavanced link layer authentication which NuttX doesn't implement yet and then fail to login the network.